From d280ed52f36c35506d423bbb63d9527048e7aab7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20Ey=C3=9Fer?= <daniel.eysser@gmail.com>
Date: Wed, 24 Apr 2019 00:09:13 +0200
Subject: [PATCH 1/2] setlocale

---
 slstatus.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git slstatus.c slstatus.c
index 96fa5b6..82420ab 100644
--- slstatus.c
+++ slstatus.c
@@ -1,5 +1,6 @@
 /* See LICENSE file for copyright and license details. */
 #include <errno.h>
+#include <locale.h>
 #include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
@@ -68,6 +69,7 @@ main(int argc, char *argv[])
 		usage();
 	}
 
+	setlocale(LC_ALL, "");
 	memset(&act, 0, sizeof(act));
 	act.sa_handler = terminate;
 	sigaction(SIGINT,  &act, NULL);
-- 
2.21.0

