From 613f477bc9002f501405209785b2b16494ae2389 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20Ey=C3=9Fer?= <daniel.eysser@gmail.com>
Date: Sat, 29 Sep 2018 02:50:46 +0200
Subject: [PATCH 7/8] ctrl-j/ctrl-k for down/up

---
 dmenu.1 | 10 ++--------
 dmenu.c | 11 ++++-------
 2 files changed, 6 insertions(+), 15 deletions(-)

diff --git dmenu.1 dmenu.1
index 323f93c..e592695 100644
--- dmenu.1
+++ dmenu.1
@@ -134,13 +134,13 @@ Backspace
 Tab
 .TP
 .B C\-j
-Return
+Down
 .TP
 .B C\-J
 Shift-Return
 .TP
 .B C\-k
-Delete line right
+Up
 .TP
 .B C\-m
 Return
@@ -148,12 +148,6 @@ Return
 .B C\-M
 Shift-Return
 .TP
-.B C\-n
-Down
-.TP
-.B C\-p
-Up
-.TP
 .B C\-u
 Delete line left
 .TP
diff --git dmenu.c dmenu.c
index 65f25ce..ce7d347 100644
--- dmenu.c
+++ dmenu.c
@@ -334,17 +334,14 @@ keypress(XKeyEvent *ev)
 		case XK_g: ksym = XK_Escape;    break;
 		case XK_h: ksym = XK_BackSpace; break;
 		case XK_i: ksym = XK_Tab;       break;
-		case XK_j: /* fallthrough */
+		case XK_j: ksym = XK_Down;      break;
+		case XK_k: ksym = XK_Up;        break;
 		case XK_J: /* fallthrough */
 		case XK_m: /* fallthrough */
 		case XK_M: ksym = XK_Return; ev->state &= ~ControlMask; break;
-		case XK_n: ksym = XK_Down;      break;
-		case XK_p: ksym = XK_Up;        break;
+		case XK_n: /* fallthrough */
+		case XK_p: /* fallthrough */
 
-		case XK_k: /* delete right */
-			text[cursor] = '\0';
-			match();
-			break;
 		case XK_u: /* delete left */
 			insert(NULL, 0 - cursor);
 			break;
-- 
2.28.0

